pajbot/models/emote.py

```diff
@@ -12,15 +12,21 @@ class Emote:
     :ivar id: Provider-specific ID, e.g. the emote ID for twitch, or the has for bttv.
     :type id: str
     :var urls: Dict mapping size (e.g. "1", "2", or "4") to an URL depicting the emote
-    :type urls: dict[str, str]"""
+    :type urls: dict[str, str]
+    :ivar max_width: The maximum width the emote can have.
+    :type max_width: int
+    :ivar max_height: The maximum height the emote can have.
+    :type max_height: int"""
 
-    def __init__(self, code: str, provider: str, id: str, urls: Dict[str, str]) -> None:
+    def __init__(self, code: str, provider: str, id: str, urls: Dict[str, str], max_width: int = 128, max_height: int = 128) -> None:
```

Comment:
You use 128 here and 112 in other places where you define a "default" - is a default value here necessary?

Reply:
I've updated the value to 112 for the sake of consistency, but as to whether I should set a default or not: I encountered issues with the bot trying to load emotes from the cache at the start, since the cached value didn't contain these properties.  So I thought I'd set a default to avoid forcing everyone to clear their emotes cache upon updating

Reply:
In that case, the cache should be cleared by this PR. This should be done using a migration revision - see `pajbot/migration_revisions/redis/0002_fix_bad_quest_emote_json.py`
you'll need to make a new revision `0003_asdasd.py` that deletes all cached emotes from Redis